js-intro.js

```diff
@@ -37,7 +62,35 @@ var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
 
+const noVowels1 = str => {
+
+    //turn string into an array
+
+    var array1 = stringWithVowels1.split('')
+
+    //create new array where the vowels are omitted. Use .filter method with "not" operator to flush out the vowels
+
+    var newArr1 = array1.filter(value => value !== 'a' && value !== 'e' && value !== 'i' && value !== 'o' && value!== 'u'
+    )
+    //now that the vowels have been removed from the array, return the array into a string with no vowels.
+
+    return newArr1.join('')
+}
+
+//repeat steps above with string2
 
+//use stringWithVowels2.toLowerCase.split('') to turn string to lowercase & included both to remove both vowels without affecting capitalization of other leters.
+
+const noVowels2 = str => {
+    var array2 = stringWithVowels2.split('')
+    var newArr2 = array2.filter(value => value !== 'a' && value !== 'e' && value !== 'I' && value !== 'i' && value !== 'o' && value!== 'u'  )
+    return newArr2.join('')
+}
```

Comment:
Same as above, consider utilizing your argument rather than a specific variable in your function. Also, you can just return line 86 and append `.join("")` to the end of the same line to save yourself some key strokes :)